static/js/common/upload-addon.js

```diff
@@ -327,12 +327,28 @@
 
                     $("<strong>").text(message).appendTo(upload_results);
 
+                    // Specific messages for unlisted addons.
+                    var isListed = $('#id_is_listed').is(':checked'),
+                        isSideload = $('#id_is_sideload').is(':checked');
+                    if (!isListed) {
+                      if (isSideload) {
+                        $("<p>").text(gettext("Your submission, as a sideloaded add-on, will go through a manual review.")).appendTo(upload_results);
```

Comment:
I don't think we're using the term 'sideloaded' in user facing pages.  Manual review is the expectation, right?  I don't think a message is required here at all..